build.gradle

```diff
@@ -125,4 +125,15 @@ task buildSconsArtifacts {
 
     dependsOn 'buildGodotOpenXRVendorsDebugGDExtension'
     dependsOn 'buildGodotOpenXRVendorsReleaseGDExtension'
+}
+
+task copyBuildToSamples {
```

Comment:
Looks good! 

Let's have this task called by the `buildPlugin` task when it completes. This can be done by using the `finalizedBy 'copyBuildToSamples'` at the end of the `buildPlugin` task definition.

Let's also include a `cleanBuildFromSamples` task that can be called by the `clean` task; i.e: the `clean` task should include a `dependsOn 'cleanBuildFromSamples'` in its definition.